src/components/Pocket/TabletPocket.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Image, Text } from 'react-native';
+import PocketIcon from '../../assets/images/PocketIcon.png';
+import Button from '../common/Button';
+import strings from '../../localization/index';
+import styles from './styles';
+
+const TabletPocket = ({ id, time, weight }) => (
+  <View style={styles.containerTabletPocket}>
+    <Image source={PocketIcon} style={styles.pocketImageStyleTablet} />
+    <View style={styles.containerItemTablet}>
+      <Text style={styles.textBlackTablet}>Id:</Text>
```

Comment:
Maybe you decided to put "Id" for some reason, but in the story image reference the word over the pocket id is "Pocket" or "Bolsón" in spanish, instead of "Id". Also it would be better if you use a string from the localization files.